workspaces/github-actions/.changeset/proud-clouds-shake.md

```diff
@@ -0,0 +1,7 @@
+---
+'@backstage-community/plugin-github-actions': patch
+---
+
+Truncate commit message to first newline.
+
+This change is to address issue https://github.com/backstage/community-plugins/issues/1246 by truncating the commit message that is displayed at the first newline character and exposes the entire message on hover.
```

Comment:
As these will get rolled up in the the Changelog for this plugin we generally keep them short. You can remove the 2nd sentence here, please. 👍